src/lwc/displayRecords/displayRecords.js

```diff
@@ -1,21 +1,41 @@
-import { LightningElement, track } from 'lwc';
+import { LightningElement,track } from 'lwc';
 import getContact from '@salesforce/apex/FetchingRecords.getContact';
 import getOpportunity from '@salesforce/apex/FetchingRecords.getOpportunity';
 import getCases from '@salesforce/apex/FetchingRecords.getCases';
+import getAccountList from '@salesforce/apex/FetchingRecords.getAccountList';
 
 export default class DisplayRecords extends LightningElement {
     listContact;
     listOpp;
     listCase;
-    idAccount;
+    @track nameAccount;
+    @track data =[];
+    @track itemList=[];
+    @track value;
+    view=false;
+    @track idAccount;
+    column=[{fieldName:'Name', label:'Name'},{fieldName:'Id', label:'Id'}];
+    caseColumn=[{fieldName:'Id', label:'Id'},{fieldName:'CaseNumber', label:'Case Number'}];
     handleClick(event){
+        this.view=true;
         var inp = this.template.querySelector("lightning-input");
-        this.idAccount = inp.value;
-        //this.idAccount=this.template.querySelector('lightning-input');
+        this.nameAccount = inp.value; 
+        getAccountList({input : this.nameAccount})
+        .then(result => {
+            let listAccounts = result;
+            for(var i=0;i<listAccounts.length;i++) {
+                this.itemList.push({label: listAccounts[i].Name, value: listAccounts[i].Id});
+            }
+            this.data = this.itemList;
```

Comment:
Also, this array is not cleared before adding new values. So it will contain the old values of the account even if you try to search with a new account name or even if you search with the same name again.